content/api/local-installation.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Local installation
+---
+
+Installing our code locally will allow your editor to extract type definitions from the `Robot` class and its boards, as well as allowing auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
+
+Installing these definitions is highly optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
+
+## Installation
+Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. On Windows, this is best done using the [`get-pip.py` script](https://pip.pypa.io/en/stable/installing).
```

Comment:
We should clarify that `pip` will be installed _if you have installed Python_. (I'm not sure whether we've already told them to do that somewhere, but even if we have I think it's worth being explicit here)

Reply:
Related to this, have we mentioned anywhere that they'll need to use Python _3_?

Reply:
It's mentioned in https://github.com/sourcebots/docs/pull/45, however i'll reitterate it here to be sure